### PR TITLE
CRAN patch and other small changes

### LINKED ISSUE #299: better docs for na.omit behavior with multiple variables
When you do `lm_robust(cbind(Y1, Y2, Y3) ~ x, data = data)` vs `lm_robust(Y1 ~ x, data = data)` when there are missing values in Y2 and Y3 that are not missing in Y1, you get different answers for effect on `Y1`. This is also the `lm` behavior. It's not necessarily what you would expect so suggest a line in the docs on this.

### PR DESCRIPTION
**Pull request by lukesonnet**
Respond to CRAN notice to fix errors:

1) Tests fail due to update in clubSandwich **(resolved)**
2) Same segfaults on Solaris in demeanMat, which I have been unable to resolve despite trying to silo the problem and rewriting the code many times **(unresolved)**

Make other small changes:

1) add note about missingness in mlm (closes #299)  **(resolved)**

### CODE REVIEW

File: R/estimatr_lm_robust.R [hunk 1]
```diff
@@ -69,6 +69,9 @@
 #' multiple fixed effect variables (e.g. if you specify both "year" and "country" fixed effects
 #' with an unbalanced panel where one year you only have data for one country).
 #'
+#' As with \code{`lm()`}, multivariate regression (multiple outcomes) will only admit
+#' observations into the estimation that have no missingness on any outcome.
+#'
```
Comment (acoppock) [inline R/estimatr_lm_robust.R]: this is great here, thank you

**Comment by lukesonnet** [comment]
Ok, I think I'll wait to see about the solaris docker container, spend maybe 2 more hours on that, and if I can't make progress then I'll push, merge, and we can resubmit to CRAN with a note. I also think that it may be worth finding someone to help me w/ #268, but I'm not sure how easy that is.

File: tests/testthat/test-iv-robust.R [hunk 2]
```diff
@@ -240,7 +240,7 @@ test_that("iv_robust matches AER + clubSandwich", {
                                         cluster = dat$clust,
                                         test = ifelse(se_type == "CR2", "Satterthwaite", "naive-t"))
 
-    cols <- if (se_type == "CR2") c("estimate", "std.error", "df", "p.value") else c("estimate", "std.error", "p.value")
+    cols <- if (se_type == "CR2") c("estimate", "std.error", "statistic", "df", "p.value") else c("estimate", "std.error", "statistic", "p.value")
```
Comment (nfultz) [inline tests/testthat/test-iv-robust.R]: Can we do something that makes it cleaner what is happening?

     cols <- setdiff(c("estimate", "std.error", "statistic", "df", "p.value"), if(se_type != 'CR2') 'df')

or something?

Comment (nfultz) [reply tests/testthat/test-iv-robust.R]: Never mind, this is test code.

**Comment by graemeblair** [comment]
Is this good to go @lukesonnet?

**Comment by lukesonnet** [comment]
I was seeing if Neal was going to take a stab at the segfault issue, as I hinted at on slack. Rhub wasn't working so I couldn't test and iterate myself.

I'll add notes about not resolving that issue and resubmit tonight unless instructed otherwise.

**Comment by nfultz** [comment]
I can take a look at the solaris issue sometime over the holiday, but I'd
suggest submitting now regardless.

**Comment by lukesonnet** [comment]
A dependency of data.table is not installing properly. Please approve so I can submit.

**Comment by lukesonnet** [comment]
Need the merging to be unblocked @graeme

**Comment by graemeblair** [comment]
unblocked sorry

**Pull request merged**